code/modules/client/client_procs.dm

```diff
@@ -1185,3 +1143,12 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 
 	var/mob/dead/observer/observer = mob
 	observer.ManualFollow(target)
+
+/client/verb/stop_client_sounds()
+	set name = "Stop Sounds"
+	set category = "OOC"
+	set desc = "Stop Current Sounds"
+	SEND_SOUND(usr, sound(null))
+	var/client/C = usr.client
+	C?.tgui_panel?.stop_music()
```

Comment:
change C to something more descriptive

Reply:
this is old, i dont even know why the hell this used client anyway considering its a verb on client

im changing it to src